setup.py

```diff
@@ -58,7 +58,7 @@
           'six >= 1.9.0',  # >= 1.9.0 required by prov
           'psutil',
           'scandir',
-          'prov == 1.5.1',
+          'prov == 1.5.1, >= 1.5.2',
```

Comment:
```suggestion
          'prov == 1.5.1, >= 1.5.3',
```
Should this be 1.5.3, to match the requirements.txt?